components/forgotpasswordflow/forgotpasswordflow.tsx

```diff
@@ -1,9 +1,9 @@
 "use client";
 import React, { useState } from "react";
-import Step1 from "./step1";
-import Step2 from "./step2";
-import Step3 from "./step3";
-import Step4 from "./step4";
+import Step1 from "./Step1";
```

Comment:
There is a routing issue here.  Update the paths correctly. Use the same file name.